kraken-build/pyproject.toml

```
@@ -94,6 +94,9 @@ enable_error_code = "ignore-without-code, possibly-undefined"
 [tool.ruff]
 line-length = 120
 
+[tool.ruff.lint]
+select = ["E4", "E7", "E9", "F", "I"]
+
```

Can you add this option and rerun `ruff` ? Kraken doesn't detect first party packages because it is executed in a different venv.
```suggestion


[tool.ruff.lint.isort]
known-first-party = ["kraken"]

```